config/initializers/paperclip.rb

```diff
@@ -0,0 +1,3 @@
+Paperclip::Attachment.default_options[:path] = '/:class/:attachment/:id_partition/:style/:filename'
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.